src/test.rs

```diff
@@ -50,6 +50,51 @@ mod tests {
         builder.connect("goggle.com", s).unwrap();
     }
 
+    #[test]
+    fn connect_no_root_certs() {
+        let builder = p!(TlsConnector::builder().disable_built_in_roots(true).build());
+        let s = p!(TcpStream::connect("google.com:443"));
+        assert!(builder.connect("google.com", s).is_err());
+    }
+
+    #[test]
+    fn server_no_root_certs() {
```

Comment:
Let's add a negative test for this behavior too.

Reply:
That already exists in `connect_no_root_certs` above, and this test (depending on which negative we're talking about):
https://github.com/sfackler/rust-native-tls/blob/3a29e956c1eadba0364b3071569aa71f9defc1ff/src/test.rs#L231

Reply:
I synced up with @cyang1 offline. It looks like `server_untrusted` is the negative test we need. This lgtm!